### PR TITLE
Review certs created by tests

### PR DESCRIPTION
**Pull request by lucafavatellawork**
TODO:
- [x] Review remaining test certs.
- [x] Get #73 merged.
- [x] Rebase this PR on master.
- [x] Move from draft to ready-for-review.

### CODE REVIEW

**Comment by lucafavatellawork** [comment]
@miklosduma I did not replace `mktemp_pathname` with `tmp_path` in `test_render_local_monitor`, feel free to have a look at that if you want.

File: sparkl_cli/test/test_put_source_rm.py [hunk 1]
```diff
@@ -141,7 +145,8 @@ def test_render_monitor(setup_test_put_source_rm):
     assert isinstance(result, types.GeneratorType)
 
 
-def test_render_local_monitor(setup_test_put_source_rm, temp_file_setup):
+def test_render_local_monitor(setup_test_put_source_rm, tmp_path):
+    temp_file = tmp_path / "render.tmp"
 
     # Create temporary file and copy Primes into it
     temp_path = mktemp_pathname(".xml")
```
Comment (lucafavatellawork) [inline sparkl_cli/test/test_put_source_rm.py:152]: I did not manage to replace this with `tmp_path`.

Comment (miklosduma) [reply sparkl_cli/test/test_put_source_rm.py]: It is replaced now, thanks

Comment (lucafavatellawork) [reply sparkl_cli/test/test_put_source_rm.py]: I saw your commit 5d34729f0590085518f3884326d6461ba9e3d1b5 - brilliant, thanks!

**Comment by lucafavatellawork** [comment]
The new commit 5d34729 looks good to me. (I tested it locally with #75 .)

**Pull request merged**